packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -147,8 +147,10 @@ class RenderableTiledMap {
 
   /// Render [batchesByLayer] that compose this tile map.
   void render(Canvas c) {
-    batchesByLayer.forEach((batchMap) {
-      batchMap.forEach((_, batch) => batch.render(c));
+    batchesByLayer.forEachIndexed((i, batchMap) {
+      if (map.layers[i].visible) {
```

Comment:
Would the index be mismatched here? The batchesByLayer list is only a list of visible layers (see _renderableTileLayers), whereas map layers is all layers. Imagine you had 2 visible layers, 1 invisible layer, then 1 visible layer. BatchesByLayer[2] would point to the invisible layer.

It's a bit less efficient, but would it make sense to require a refreshCache call for every change, and just render the batchesByLayer list without any checks?

Reply:
You are right! It's also much easier and rendering is more efficient because the `if` is not needed. I have changed it.
Visibility can now be changed as follows:
```dart
var tl = tiles.tileMap.map.layers[x] as TileLayer;
tl.visible = false;
tiles.tileMap.refreshCache();
```

Reply:
Can you add a convenience function for it too?
Just something like:
```dart
void changeVisibility(int layerId, bool visibility) {}
```